### DISCUSSION TITLE
Highlight caution zones on waveform

### DISCUSSION THREAD

**Discussion by tharsim** [SubtitleEdit/subtitleedit#5889] [Ideas]
I was working and thought of this:
[https://i.imgur.com/ibLGyJa.jpg](https://i.imgur.com/ibLGyJa.jpg)

Not sure if it's applicable. The idea is to highlight (before and after each scene change) a certain amount of frames to determine the caution zone as required by most style guidelines in the industry. I draw a 12-frame example in the image, corresponding to half a second on videos that use 24 FPS. The amount of frames to highlight would be selected along with the option "Generate/import scene changes...". It's purely a visual aid.

**Comment by niksedk** [comment]
![image](https://user-images.githubusercontent.com/1261331/162262407-8ea4b575-5b32-47ef-8339-c72fbaa36eb5.png)
